src/api/controllers/auth.controller.ts

```diff
@@ -0,0 +1,68 @@
+import { Request, Response } from 'express';
+import * as jwt from 'jsonwebtoken';
+import { Service } from 'typedi';
+
+import { Config } from '../../config';
+import { HandleError, LogRequestData } from '../../decorators';
+import { RefreshTokenService, UserService } from '../../services';
+import { HttpCode, HttpMessage } from '../constants';
+
+@Service()
+export class AuthController {
+    constructor(private userService: UserService, private refreshTokenService: RefreshTokenService) {}
+
+    @LogRequestData()
+    @HandleError()
+    async login(req: Request, res: Response) {
+        const { login, password } = req.body;
+        const user = await this.userService.getUserByLogin(login);
+
+        if (!user) {
+            return res.status(HttpCode.NotFound).send(HttpMessage.NotFound);
+        }
+
+        if (password !== user?.password) {
+            return res.status(HttpCode.Forbidden).send(HttpMessage.Forbidden);
+        }
+
+        const accessToken = jwt.sign({ sub: user.id }, Config.auth.secret, { expiresIn: Config.auth.jwtExpiration });
+        const refreshToken = await this.refreshTokenService.createToken(user.id);
+
+        if (!refreshToken) {
+            return res.status(HttpCode.InternalServerError).send(HttpMessage.InternalServerError);
```

Comment:
JFYI, there is [`sendStatus`](https://expressjs.com/en/4x/api.html#res.sendStatus) method

Reply:
@KonanMentor fixed